test/parallel_api/numeric/numeric.ops/scan.pass.cpp

```diff
@@ -27,69 +28,6 @@
 
 using namespace TestUtils;
 
-// We provide the no execution policy versions of the exclusive_scan and inclusive_scan due checking correctness result of the versions with execution policies.
```

Comment:
Availability the serial scanning algorithms in the C++ standard library depends on C++ version and version of the standard library. We should check these  versions and depending on that enable these serial versions of scan or don't - see //TODO on 31th line.

Reply:
OK, will fix

Reply:
We already have self-implementer functions like inclusive_scan_serial, exclusive_scan_serial in scan.pass.cpp
I simply extract them to new header file.